test/e2e/es/stackconfigpolicy_test.go

```diff
@@ -57,6 +58,8 @@ func TestStackConfigPolicy(t *testing.T) {
 
 	namespace := test.Ctx().ManagedNamespace(0)
 	secureSettingsSecretName := fmt.Sprintf("test-scp-secure-settings-%s", rand.String(4))
+	secretMountsSecretName := fmt.Sprintf("test-scp-secret-mounts-%s", rand.String(4))
```

Comment:
Let's maybe ensure that these 2 Secrets are eventually deleted?

Reply:
I will add a step to delete these secrets during cleanup, these are mimicking user created secrets, so will not be deleted by the operator itself